BagTrackTests/BagsTableViewControllerTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  BagsTableViewControllerTests.swift
+//  BagTrackTests
+//
+//  Created by Robin Kipp on 16.11.17.
+//  Copyright © 2017 Robin Kipp. All rights reserved.
+//
+
+import XCTest
+import CoreLocation
+@testable import BagTrack
+
+class BagsTableViewControllerTests: XCTestCase {
+
+    var bundle:Bundle!
+    var storyboard:UIStoryboard!
+    var subject:BagsTableViewController!
+    var bag:Bag!
+    var dataModel:DataModel!
+
+    override func setUp() {
+        super.setUp()
+        bundle = Bundle(for: BagsTableViewController.self)
+        storyboard = UIStoryboard(name: "Main", bundle: bundle)
+        subject = storyboard.instantiateViewController(withIdentifier: "MyBags") as! BagsTableViewController
+        let testUUID = UUID(uuidString: "39EF51FA-F162-4A11-8DF4-F636D20679F8")!
+        let major = CLBeaconMajorValue(1)
+        let minor = CLBeaconMinorValue(1)
+        bag = Bag(name: "Test Bag", proximityUUID: testUUID, majorValue: major, minorValue: minor, beaconID: "TestBag123")
+        dataModel = DataModel.sharedInstance
+    }
+
+    override func tearDown() {
+        subject = nil
+        storyboard = nil
+        bundle = nil
+        bag = nil
+        dataModel = nil
+        super.tearDown()
+    }
+
+    func testBagStates() {
+        XCTAssertNotNil(bag)
+        XCTAssertEqual(bag.proximity, .unknown)
+        bag.proximity = .immediate
```

Comment:
I'm not sure how valuable it is to test these. For one thing, they're localized strings, and you're just testing the English values. For another thing, there's not really any logic that you're testing here. It's what Andy Matuschak would call "XML-like code" - that is, it doesn't really have much logic that's worth testing. It's just a list of mapped values.

What might be worth testing is if your `proximityForDisplay()` function returned an enum, and the enum returned a localized `displayString` for the user. That way, you could test that the right enum value is returned for the right proximity value, and separate that logic from the logic of showing the right localized string.